team/team_test.go

```diff
@@ -84,9 +84,11 @@ name = "Kiffix"
 		})
 
 		t.Run("sets a valid signature", func(t *testing.T) {
-			verifyRosterSignature(t,
```

Comment:
suspect you could also drop `func verifyRosterSignature()` now?

Reply:
Ah great spot, I missed out that commit from this PR!

Reply:
Actually I just forgot to delete it!

Reply:
Done